### ISSUE TITLE
Limit file attachment and site archive size

### ISSUE THREAD

**Issue by meldarionqeusse** [freetonik/textpod#26]
I noticed the when archiving a web page like a wikipedia entry I ended up with a 200MB site which took a while to create and takes a while to load. This lead me to think that user should be at least warned about the file size if not limit upload sized to a specified amount.

**Comment by freetonik** [comment]
Good idea, thanks for reaching out. Though it's difficult to predict the size beforehand. This needs some thought. Say, Monolith saves the file and it's over the limit. Should it delete it and inform the user? Feels a bit wasteful. Perhaps, the limits should be solely for information purposes, and the user should be able to delete the attachment easily.
